### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by bagibest**

### CODE REVIEW

**Review by stanislav-topikha — CHANGES_REQUESTED** [review]
checklist 
https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md

File: src/transformState.js [hunk 1]
```diff
@@ -4,8 +4,25 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    if (actions[i].type === 'addProperties') {
+      for (const action in actions[i].extraData) {
```
Comment (AlisaVlasova) [inline src/transformState.js:11]: image

File: src/transformState.js [hunk 2]
```diff
@@ -4,8 +4,25 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    if (actions[i].type === 'addProperties') {
```
Comment (AlisaVlasova) [inline src/transformState.js:10]: image

File: src/transformState.js (hunk 1 above)
Comment (AlisaVlasova) [inline src/transformState.js:11]: image

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]

**Pull request closed without merging**